modules/ROOT/pages/tools/io-tool/filesending.adoc

```diff
@@ -0,0 +1,46 @@
+:imagesdir: 
+= File Sending
+
+When you share, you can choose to publish or send the files. Publishing means you send them out and anyone who is interested and has access can download them. On the other hand, when you send the files, you select a specific recipient.
+
+image::io-tool/sendorpublish.png[]
+
+Before sending the files you need to upload them to your _Data Storage_ (drag and drop, or click to choose). This is a pop up message you receive when uploading a valid data.
+
+image::io-tool/multifieldsupload.png[]
+
+When sending files, you can only select thoes from your _Data Storage_. 
+
+image::io-tool/sending.png[]
+
+
+If all goes well, you will receive the following pop-up message in the top right corner.
+
+image::io-tool/sentsuccess.png[]
+
+To receive the sent file, you will then need to switch to the recipient's IO-Tool account and select the _Data Storage_ section of the main menu. You can then choose between
+
+* Receive all
+* Receive from today
+* Clear feed
+
```

Comment:
The Receive all and Receive from today-Buttons are only needed, if PushNotifications are not active.

The Clear feed button is specifically useful in case you sent a broken dataset to the IO-Tool (imagine a farming software developer send a ZipFile in a broken format so that the IO-Tool cannot load it. To avoid future errors, it's better to clear the feed.) You should point out and warn, that this button will delete all messages from the Feed in agrirouter for this specific endpoint and that there's no additional popup to warn the user.